### DISCUSSION TITLE
Temperature graph

### DISCUSSION THREAD

**Discussion by damianoski** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#778] [Ideas]
Hello,

Thank you very much for this amazing project.

I'm not sure if the hardware still has available resources to include this, but it would be fantastic to have a simple temperature graph 📈 with a short history of 24h, 3days, if posible also 7 days. In resolution of 0.5 - 1h.

The temperature graph will help with understanding temperature interactions:

- Determining how much time is needed to heat up the hot tub to the target temperature based on the specific hot tub and ambient temperature, using previous heating data.
- How quickly the temperature is dropping.
- What the best standby temperature is.

Kind Regards
Damianoski

**Comment by metaljay** [comment]
A lot of people run this via home assistant which has this capability natively. There isn’t much space left on the ESP to store historical data.

**Reply by visualapproach** [reply]
Thanks for your suggestion! As @metaljay said. It may or may not make the esp crumble. I'll keep it in mind, even though these kind of features is much better suited for a home automation systems.

**Reply by damianoski** [reply]
Thank you for your interest in this topic.

Regarding the graph: Even if it is possible to save 24 hours of temperature data to a .csv, xml file that can be downloaded and if needed opened in Excel, that would be good enough for the beginning.
